src/hooks/users/useRecentWorldUsers.ts

```diff
@@ -1,30 +1,44 @@
-import { useMemo } from "react";
+import { isLoaded } from "react-redux-firebase";
+import { isEqual } from "lodash";
 
-import { User } from "types/User";
+import { User, userWithLocationToUser } from "types/User";
 
 import { WithId } from "utils/id";
+import { worldUsersSelector } from "utils/selectors";
 import { normalizeTimestampToMilliseconds } from "utils/time";
 
+import { useSelector } from "hooks/useSelector";
+import { useConnectWorldUsers } from "hooks/users";
 import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
 
-import { useWorldUsers } from "./useWorldUsers";
+const noUsers: WithId<User>[] = [];
```

Comment:
```suggestion
const noUsers: never[] = [];
```

Reply:
As per https://github.com/sparkletown/sparkle/pull/1703#discussion_r666009006, nope, because we want it to have the correct type.